src/main/java/net/neoforged/neoforge/common/extensions/IBlockExtension.java

```diff
@@ -1019,4 +1019,8 @@ default BubbleColumnDirection getBubbleColumnDirection(BlockState state) {
             return BubbleColumnDirection.NONE;
         }
     }
+
+    default boolean shouldHideAdjacentFluidFace(BlockState state, FluidState adjacentFluid) {
```

Comment:
This should have the same docs as the `BlockState`-level method and both should ideally have some short parameter docs.